www/settings/custom_metrics/securityHeaders.js

```diff
@@ -0,0 +1,117 @@
+const data = $WPT_REQUESTS;
+const HEADERS_SCORES = {
+  "strict-transport-security": 25,
+  "content-security-policy": 25,
+  "x-frame-options": 20,
+  "x-xss-protection": 20,
+  "x-content-type-options": 20,
+  "public-key-pins": 30,
```

Comment:
Wasn't HPKP frowned upon for causing more harm than good and served as a major footgun for sites that tried to implement it? I guess it's possible to get 100 without it but it feels strange that it gets so many points.

Reply:
Overall agree but Scott's FAQ pages which details the score here https://scotthelme.co.uk/scoring-transparency-on-securityheaders-io/ still references it. I pinged him to verify that but no response yet. There's no specific obligation to be in par with his website but I figured it's a good benchmark in-case people compare and will save them the trip to securityheaders.com for an additional check if we include the same checks and scores.

I'm happy to commit future updates if Scott updates me on any new grades :-)